charts/sonarqube/templates/networkpolicy.yaml

```diff
@@ -99,4 +99,7 @@ spec:
 {{- with .Values.networkPolicy.additionalNetworkPolicys -}}
 {{ toYaml . | nindent 2 }}
 {{- end }}
+{{- with .Values.networkPolicy.additionalNetworkPolicies -}}
```

Comment:
In this case, if we use both ways to add network policies, we end up with having the unsupported network policy and the support ones both added.

Shouldn't we "disable" `additionalNetworkPolicys` when `additionalNetworkPolicies` is used?

Reply:
i thought about it, my reasoning so far is that the template is clean, no if else. In the end we are talking about adding stuff, so they can be added simultaneously it will not interfer with each other as its just a typo, and from the user need, they will very likely use one or the other.

But yeah maybe forcing the disable would be better I don't have a super strong opinion here

Reply:
I think we should be coherent with what we do elsewhere. If the new feature is used, the deprecated one is disabled...WDYT?